acceptance_tests/features/step_definitions/DSP-92_.steps.rb

```diff
@@ -32,5 +32,5 @@
 
 Then(/^I am on Step Three of the Lost\/Stolen form$/) do                                                
   page.should have_content('Step 3 of 5')                                               
-  page.should have_content('What are your personal details?')
+  page.should have_content('What are your personal details')
```

Comment:
This probably should have a question mark. Have I removed the question mark from the code?